manager/controllers/app/suite_test.go

```diff
@@ -100,6 +101,7 @@ var _ = BeforeSuite(func(done Done) {
 		Expect(err).ToNot(HaveOccurred())
 		err = NewBlueprintReconciler(mgr, "Blueprint", fakeHelm).SetupWithManager(mgr)
 		Expect(err).ToNot(HaveOccurred())
+		SetupPlotterController(mgr, local.NewManager(mgr.GetClient(), "m4d-system"))
```

Comment:
@shlomitk1 Part of this PR?

Reply:
The tests did not test what was expected. I can split into two PRs.